### DISCUSSION TITLE
Possible to fuzzy/regex filter when opening URLs?

### DISCUSSION THREAD

**Discussion by dantleech** [qutebrowser/qutebrowser#7537] [Ideas] [answered]
Hi!

Currently when using `:open` I get autocomplete for substring matches, but often I have URLs that require two matches:

```
mything.foobar.staging.com/foo
mything.foobar.staging.com/bar
mything.foobar.staging.com/baz
mything.foobar.production.com/foo
```

in the above I'd want to be able to type either `mythprod` (fuzzy) or `myth.*prod` (regex)

Is this already possible in some way, would it be a valid feature request/PR candidate?

Thanks for the amazing work on this project!

Dan

**Comment by mschilli87** [comment, marked as answer]
AFAIR `myth prod` would match `mything.foobar.production.com/foo` (only) in that case.

**Comment by dantleech** [comment]
oh my! all these years. thanks :sweat_smile:
